lesson_24/kimberlees-web-server/index.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const path = require("path");
+const app = express();
+const port = 4000;
+const cors = require("cors"); 
+
+app.use(express.urlencoded({extended: true}));
+app.use(express.json());
+app.use(express.static(path.join(__dirname, 'public')));
+app.use(cors());
+
+app.get("/", async (req, res) => {
+    res.send("This is working")
+});
+
+
+
+app.listen(port, () => {
+    console.log('Listening at http://localhost:${port}');
```

Comment:
Should use backticks (`) instead of single quotes ('). Then the port will print correctly using string interpolation.

See more here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals